js/web/lib/wasm/wasm-core-impl.ts

```diff
@@ -237,18 +235,7 @@ export const createSession = async(
 
     const [inputCount, outputCount] = getSessionInputOutputCount(sessionHandle);
 
-    let graphCaptureEnabled = false;
-    if (currentEpName === 'webgpu') {
-      const executionProviders = options?.executionProviders;
-      for (const ep of executionProviders!) {
-        const epName = typeof ep === 'string' ? ep : ep.name;
-        if (epName === 'webgpu') {
-          const webgpuOptions = ep as InferenceSession.WebGpuExecutionProviderOption;
-          graphCaptureEnabled =
-              webgpuOptions.graphCaptureEnabled === undefined ? false : webgpuOptions.graphCaptureEnabled;
-        }
-      }
-    }
+    const enableGraphCapture = options?.enableGraphCapture === undefined ? false : options.enableGraphCapture;
```

Comment:
nit: use `const enableGraphCapture = !!options?.enableGraphCapture` should be good